### DISCUSSION TITLE
step (Windows) with Silentflag

### DISCUSSION THREAD

**Discussion by stastka** [smallstep/certificates#856] [Ideas] [answered]
I create a Workflow to generate Certificate for WinRM (Windows Remote Management) over HTTPS with Powershell. It's little bit Tricky but was successful. 

I run the Workflow over Invoke-Command (Remote Execution) and the Output from step (Sample: ✔ Provisioner abc...) return to my Shell a False Positive Error. I make a workaround with save Token to a Textfile but not so elegant. 
![stepca-powershell](https://user-images.githubusercontent.com/6085488/158836220-58188931-4580-40f8-85e5-700f3123568a.png)

my wish: a silent Flag --silent to reduce output.
Dany

**Comment by maraino** [comment, marked as answer]
Hi @stastka, I'll transfer this to an issue.

**Reply by maraino** [reply]
Transferred to https://github.com/smallstep/cli/issues/651
